src/ui/public/vislib/__tests__/visualizations/column_chart.js

```diff
@@ -70,7 +70,7 @@ dataTypesArray.forEach(function (dataType, i) {
       });
 
       it('should stack values', function () {
```

Comment:
I think it would be better to break this test into two separate tests.  One for `should stack values when mode is stacked` and the other for `should stack values when mode is percentage`.  This way if one of them regresses, we can quickly identify which one.